### ISSUE TITLE
Odd dependency

### ISSUE THREAD

**Issue by renich** [error10/bitcoin-dist#9]
Hello,

On Fedora 20, I get the following error due to a strange dependency:

```
...
--> Finished Dependency Resolution
Error: Package: bitcoin-server-0.8.6-2.fc20.x86_64 (bitcoin)
       Requires: file:///usr/share/doc/selinux-policy/html/index.html
You could try using --skip-broken to work around the problem
You could try running: rpm -Va --nofiles --nodigest
```

**Comment by error10** [comment]
That is strange... I don't have time to look at it tonight but will do so tomorrow. In the meantime please try manually installing the `selinux-policy-doc` package, and then trying to install `bitcoin-server` again. Please report if this works or if it fails.

```
yum install selinux-policy-doc
yum install bitcoin-server
```

**Comment by renich** [comment]
done and the problem persists.

**Comment by error10** [comment]
This should be resolved in 0.8.6-3, which is now hitting the repos.

**Comment by renich** [comment]
On Wed, Jan 8, 2014 at 12:22 AM, error10 notifications@github.com wrote:

> This should be resolved in 0.8.6-3, which is now hitting the repos.

thanks

## 

It's hard to be free... but I love to struggle. Love isn't asked for; it's
just given. Respect isn't asked for; it's earned!
Renich Bon Ciric

http://www.woralelandia.com/
http://www.introbella.com/
